rosdep/base.yaml

```diff
@@ -965,6 +965,12 @@ gksu:
 glc:
   arch: [glc]
   gentoo: [media-libs/quesoglc]
+glfw3:
```

Comment:
please rename the key to `libglfw3-dev` as per initial review (note that the block will need to be moved to the appropriate location to respect alphabetical order)

Reply:
Done, sorry for the misunderstanding.